documentation/book/cluster-operator.adoc

```diff
@@ -1060,111 +1060,121 @@ They will not be added to the Docker image from a previous build.
 To add multiple plugins to the deployment, they all have to be added within the same build.
 
 
-=== Provisioning Role-Based Access Control (RBAC) for the operator
+=== Provisioning Role-Based Access Control (RBAC) for the Cluster Operator
+
+For the operator to function correctly it needs permission within the {ProductPlatformName} cluster to interact with the resources it manages and interacts with. 
+These include the desired resources, such as  `Kafka`, `Kafka Connect` and so on, as well as the managed resources, such as ConfigMaps, Pods, Deployments, StatefulSets, Services and so on.
+Such permission is described in terms of {ProductPlatformName} role-based access control (RBAC) resources:
+
+* ServiceAccount,
+* Role and ClusterRole,
+* RoleBinding and ClusterRoleBinding.
+
+As well as running under its own ServiceAccount with ClusterRoleBindings, the Cluster Operator itself manages some RBAC resources.
+This is because some of the components that are deployed need access to {ProductPlatformName} resources. 
+Furthermore, {ProductPlatformName} includes privilege escalation protections which prevent components operating under one ServiceAccount from granting other ServiceAccounts privileges they do not have. 
+This means that for the Cluster Operator to be able to create the ClusterRoleBindings and RoleBindings needed by resources it manages it needs those same privileges itself.
+
+[[delegated_privileges]]
+==== Delegated Privileges
+
+When the Cluster Operator deploys resources for a desired `Kafka` resource it also creates ServiceAccounts, RoleBindings and ClusterRoleBindings, as follows:
+
+* The Kafka broker pods use a ServiceAccount called `[cluster-name]-kafka` (where `[cluster-name]` is a placeholder for the name of the Kafka resource)
+  - When the rack feature is used, the `strimzi-[cluster-name]-kafka-init` ClusterRoleBinding is used to grant this ServiceAccount access to the nodes within the cluster via a ClusterRole called `strimzi-kafka-broker`
+  - When the rack feature is not used no binding is created.
+* The Zookeeper pods use the default ServiceAccount as they have no need to access {ProductPlatformName} resources
+* The Topic Operator pod uses a ServiceAccount called `[cluster-name]-topic-operator` (where `[cluster-name]` is a placeholder for the name of the Kafka resource)
+    - The TopicOperator produces {ProductPlatformName} events with status information, so the ServiceAccount is bound to a `ClusterRole` called `strimzi-topic-operator` which grants this access via the `strimzi-topic-operator-role-binding` RoleBinding.
```

Comment:
Will it always be strimzi- or will it use the product name?

Reply:
It will always be strimzi, right @scholzj?